### DISCUSSION TITLE
Clarification on JSON requirements

### DISCUSSION THREAD

**Discussion by zprice22** [CMSgov/hospital-price-transparency#18] [Q&A] [answered]
In the JSON data attributes table (https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation/JSON) the license_information is noted as required. The hospital licensure object detail indicates that the license_number is not required. And that only the state is required. Please clarify the difference.

Additional inconsistencies between the attributes table and the objects are also noted for standard charge information, drug information and modifiers.

**Comment by RidenourK** [comment, marked as answer]
@zprice22 –Thank you for your feedback. You’re correct in that the license_information object is required on the root node of the JSON schema. The reason that the license_number attribute in the license_information object isn’t required is due to rare situations where a hospital may not have been issued a license number by the state. Therefore, requiring the state attribute in the object is the minimum requirement, however, if a hospital has a license number, they must include it in their MRF in the license_number attribute.

Similarly, for the other objects mentioned, the object may not be relevant for every item and service in the MRF and is therefore not required. However, if that object is created then certain attributes in the object may be required. For example, the drug_information object is not required for every item and service, but if this object is created then both the unit and type attributes are required for this data to be complete and understandable by the end user.

Please let us know where you see additional inconsistencies and hopefully, we can clear up the confusion.
